### ISSUE TITLE
Exception when changing parent of new page

### ISSUE THREAD

**Issue by rhukster** [getgrav/grav-plugin-admin#2018]
Created a new page, and it was not saved yet.  I then changed parent, and hit save. This lead to an exception:

```
TypeError
Argument 1 passed to Grav\Framework\Route\RouteFactory::createFromString() must be of the type string, null given, called in /a/b/c/grav/grav/system/src/Grav/Common/Flex/Types/Pages/PageObject.php on line 103
```

**Comment by rhukster** [comment]
A quick note, I had to back to pages list and then i noticed the move of the page into the correct location had occurred. Although order was not fixed, still shows `9999999`.

Saving again fixed the ordering.

**Comment by mahagr** [comment]
Fixes so far:
https://github.com/getgrav/grav/commit/57c6414d1c7fed10ed9f7b0d4458da35c7a1b7ac
https://github.com/trilbymedia/grav-plugin-flex-objects/commit/e81367492654da61dc847fae8a0ef66e08a7457b
